app/views/offers/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for [@offer] do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :price, as: :integer %>
+  <%= f.input :number_of_people, as: :integer %>
+  <%= f.input :category %>
```

Comment:
this should be a dropdown of available options. Categories are available in the model - offer.rb

Reply:
This is the form from Johanna - I just put it in a different file. @johanna-maier - can you please update it?

Reply:
Yes, will take a look tomorrow